vcsclient/github.go

```diff
@@ -524,3 +568,32 @@ func mapGitHubPullRequestToPullRequestInfoList(pullRequestList []*github.PullReq
 	}
 	return
 }
+
+func convertToBase64(data []byte) string {
+	sEnc := b64.StdEncoding.EncodeToString(data)
+	return sEnc
+}
+
+func packScanningResult(data []byte) (string, error) {
+	compressedScan, err := compressPackage(data)
+	if err != nil {
+		return "", err
+	}
+	packScan := convertToBase64(compressedScan)
+
+	return packScan, nil
+}
+
+func compressPackage(stream []byte) ([]byte, error) {
+	var b bytes.Buffer
+	w := gzip.NewWriter(&b)
```

Comment:
let's move the close statement to here (inside a differ call)

Reply:
done